src/StudentsController.ts

```diff
@@ -230,6 +263,10 @@ export class StudentsController {
             }
         }
 
+        if (!student.correspondingRelationshipTemplateId) {
+            status = "deleted";
+        }
```

Comment:
Is it save to assume that if there is no template id, the student is deleted? Is there no other case where this condition might be true?